src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -24,7 +25,7 @@ export class MdTabNavBar {
 @Directive({
   selector: '[md-tab-link]',
 })
-export class MdTabLink {
+export class MdTabLink extends MdRipple {
```

Comment:
Conceptually, I find it a bit confusing that the link extends a ripple.  Any reason why we wouldn't convert this to a component and use the md-ripple-trigger property?